exporter/dynatraceexporter/metrics_exporter_test.go

```diff
@@ -52,7 +51,7 @@ func Test_exporter_PushMetricsData(t *testing.T) {
 			Invalid: 0,
 		}
 		body, _ := json.Marshal(response)
-		w.Write(body)
+		_, _ = w.Write(body)
```

Comment:
should we be checking these errors as part of the test?  Same question elsewhere.